azure/src/main/java/com/microsoft/azure/management/Azure.java

```diff
@@ -666,6 +671,20 @@ public BatchAccounts batchAccounts() {
         return batchManager.batchAccounts();
     }
 
+    /**
+     * @return entry point to managing batch AI clusters.
+     */
+    public BatchAIClusters batchAIClusters() {
+        return batchAIManager.clusters();
+    }
+
+    /**
+     * @return entry point to managing batch AI file servers.
+     */
+    public FileServers fileServers() {
+        return batchAIManager.fileServers();
```

Comment:
The name  `fileServers()` is very generic. Since this is roll up client, can we be more specific here like `batchAIFileServers()` or some name which avoid confusion when other RP also uses the term `FileServers` in future.